edge-modules/functions/samples/docker/linux/arm32v7/Dockerfile

```diff
@@ -1,8 +1,6 @@
-ARG buildno
-FROM edgebuilds.azurecr.io/microsoft/azureiotedge-functions-binding:${buildno}-linux-arm32v7
+FROM mcr.microsoft.com/azure-functions/dotnet:2.0-arm32v7
 
-WORKDIR /app
+ENV AzureWebJobsScriptRoot=/app
 
-COPY samples/ samples/
+COPY . /app
```

Comment:
Should we copy to /approot (and not set AzureWebJobsScriptRoot) here, to be consistent between Linux and Windows?

Reply:
In Linux container the default is value for AzureWebJobsScriptRoot is /home/site/wwwroot